leafletmap/src/test/kotlin/de/saring/leafletmapdemo/Controller.kt

```diff
@@ -65,12 +65,27 @@ class Controller {
         setupPositionSliderRange()
     }
 
+    @FXML
+    private fun onClear() {
+        mapView.clearMarkerAndPaths()
+
+        resetPositionSliderRange()
+        positionMarkerName = null
+        positionTooltip.hide()
+    }
+    
     private fun setupPositionSliderRange() {
         slPosition.min = 0.0
         slPosition.max = track.positions.size - 1.0
         slPosition.value = 0.0
     }
 
+    private fun resetPositionSliderRange() {
+        slPosition.min = 0.0
```

Comment:
The min value stays 0, don't needed here.

Reply:
Done